pygfunction/ground_heat_exchanger.py

```diff
@@ -0,0 +1,67 @@
+from typing import List, Union, Dict
+
+import numpy.typing as npt
+
+from .borefield import Borefield
+from .enums import PipeType
+from .media import Fluid
+from .networks import Network
+from .pipes import get_pipes
+
+
+class GroundHeatExchanger:
+
+    def __init__(self,
+                 H: npt.ArrayLike,
+                 D: npt.ArrayLike,
```

Comment:
New GHE class. Constructor and `evaluate_g_function` arguments need a more thorough review.